### DISCUSSION TITLE
Format of output filename. Is is possible to change?

### DISCUSSION THREAD

**Discussion by tekstacey** [aclap-dev/video-downloadhelper#16] [Help and Q&A] [answered]
### Your question

I would like to know how to change the numbering of a series of downloads.
Currently it starts numbering like 'Name", then "Name1", then "Name2".

Is it possible to pad the numbering and specify the starting count?

i.e., "Name01", "Name02"  or even "Name-01", "Name-02"

I have looked through all of the setting available and must be missing something.
I did not find any other reference to this in the discussions.

### Addon & CoApp version and details

version: 8.1.4.0
beta: false
build date: 2024-01-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\sbrewer",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by paulrouget** [comment]
> Is it possible to pad the numbering and specify the starting count?

Yes, that makes sense. I'll work on that!

**Comment by paulrouget** [comment, marked as answer]
In master. This will be part of the version 2.0.15 of the CoApp (release date not known yet - guessing in a couple of weeks).

**Comment by tekstacey** [comment]
I appreciate your work on this.

**Reply by paulrouget** [reply]
Let me know if that works for you: https://github.com/aclap-dev/video-downloadhelper/discussions/35

**Reply by tekstacey** [reply]
Not clear how to set this. The first file is always un-numbered which makes sense of you don't know it will be part of a series. I do see 2 digit numbers, but is that a fixed padding? I don't see any way to set from the client side.

**Reply by GrampaWildWilly** [reply]
#812 seems closely related to this one.

**Reply by rihntintin** [reply]
There is 2 ways to download video in VDH: (1) coApp and (2) browser. @tekstacey are you using the browser? I don't have time to experiment, but I believe the browser creates filenames with a single digit in sequence. I only use the coApp, so I know that it uses 2 digit sequence at the end of the filename. I hope this helps.

**Comment by tekstacey** [comment]
using coapp. I get the 2 digit numbering.
